### ISSUE TITLE
Moves that change type with Items display "Normal" typing

### ISSUE THREAD

**Issue by katykat5099** [rh-hideout/pokeemerald-expansion#3587]
### Description

Moves that are normally Normal type by default but can be changed in battle with an item, such as Judgment, Multi-attack, Technoblast.. remain as Normal type in the move select screen rather than its new type.
![pokeemerald-0](https://github.com/rh-hideout/pokeemerald-expansion/assets/12654608/47049064-d9ed-4ff0-a367-cfb5754e9a94)

### Version

upcoming (Edge)

### Upcoming Version

42076897d3c77e461b9b181be482c610d5981a94

### Discord contact info

kakarikiki

**Comment by Bassoonian** [comment]
Isn't that how it works in Vanilla? Hidden Power also doesn't reflect the actual type. [This YouTube video](https://www.youtube.com/watch?v=vb5D5j45Eqg) shows a Chill Drive Genesect using an Ice-type Technoblast (which is still shown as Normal in the move selection screen).

**Comment by katykat5099** [comment]
> Isn't that how it works in Vanilla? Hidden Power also doesn't reflect the actual type. [This YouTube video](https://www.youtube.com/watch?v=vb5D5j45Eqg) shows a Chill Drive Genesect using an Ice-type Technoblast (which is still shown as Normal in the move selection screen).

Ahh, you know what, I think you're right. My bad! 😆
I just double checked the other moves as well.

**Comment by PCG06** [comment]
I mean, it could be useful, right? 👀

**Comment by Bassoonian** [comment]
There's a lot of things that could be useful, but GF be GFing

**Comment by kittenchilly** [comment]
> Isn't that how it works in Vanilla? Hidden Power also doesn't reflect the actual type. [This YouTube video](https://www.youtube.com/watch?v=vb5D5j45Eqg) shows a Chill Drive Genesect using an Ice-type Technoblast (which is still shown as Normal in the move selection screen).

Actually, I think SV changed this. I know for certain a move like Ivy Cudgel changes the displayed type depending on the item.

**Comment by Bassoonian** [comment]
You're correct, [this video](https://www.youtube.com/watch?v=kLugImn1ZUA) shows Ivy Cudgel reflecting the type. We'd need to verify if that makes Ivy Cudgel the anomaly or if SV (or its DLC) did indeed make Judgment (and presumably Technoblast, Multi-Attack and possibly through analogy in expansion Hidden Power) show the type it will be, both in summary and in battle.
